pkg/devspace/deploy/deployer/kubectl/kubectl.go

```diff
@@ -74,6 +77,16 @@ func New(ctx devspacecontext.Context, deployConfig *latest.DeploymentConfig) (de
 		manifests = append(manifests, manifest)
 	}
 
+	if deployConfig.Kubectl.InlineManifest != "" {
+		// save created manifest to file
+		// fmt.Println(deployConfig.Kubectl.InlineManifest)
+		err := os.WriteFile(Cachemanifest, []byte(deployConfig.Kubectl.InlineManifest), 0644)
```

Comment:
This can cause problems with multiple deployments using the same file at the same time, please try to use an approach without a temporary file and pass in the bytes directly

Reply:
Yes about this, it was one point where I wanted to think more

This way, we're recycling the current `Deploy()` without changes
Right now we're just using the `DeployConfig.Manifests` 

One approach could be expanding the `DeployConfig` struct, with the inline manifest data, and do a specific apply for them

This would also unlock the possibility of having both manifests (paths) and inline manifests

What do you think?

Reply:
I'm not sure if having `inlineManifests` and `manifests` together is a good idea, as the order would be undefined and its easy to just define 2 deployments which would deploy those